File: CON BDD/EUREKABANK_REST_DOTNET_GR06/08. SERVIDOR/Models/Usuario.cs
namespace SERVIDOR.Models;

public class Usuario
{
    public string CodigoEmpleado { get; set; } = string.Empty;
    public string UsuarioNombre { get; set; } = string.Empty;
    public string Clave { get; set; } = string.Empty;
    public string Estado { get; set; } = string.Empty;
    public string? ClienteCodigo { get; set; }
}

File: CON BDD/EUREKABANK_RESTFULL_JAVA_GR06/08. SERVIDOR/eurekabank_restful_java_gr06/src/main/java/ec/edu/monster/modelo/Usuario.java
package ec.edu.monster.modelo;

/** Representa una fila de la tabla `usuario`. */
public class Usuario {

    private String codigoEmpleado;
    private String usuario;
    private String clave;      // hash SHA1 almacenado
    private String estado;     // ACTIVO / ANULADO
    private String clienteCodigo; // null = no es cliente (admin/empleado)

    public String getCodigoEmpleado() { return codigoEmpleado; }
    public void setCodigoEmpleado(String codigoEmpleado) { this.codigoEmpleado = codigoEmpleado; }

    public String getUsuario() { return usuario; }
    public void setUsuario(String usuario) { this.usuario = usuario; }

    public String getClave() { return clave; }
    public void setClave(String clave) { this.clave = clave; }

    public String getEstado() { return estado; }
    public void setEstado(String estado) { this.estado = estado; }

    public String getClienteCodigo() { return clienteCodigo; }
    public void setClienteCodigo(String clienteCodigo) { this.clienteCodigo = clienteCodigo; }
}
